registration-processor/pom.xml

```diff
@@ -107,11 +107,11 @@
 		<jsch.version>0.1.55</jsch.version>
 

```

Comment:
In this file, i was able to see the below.

<object.store.khazana.version>1.1.5.5-rc2</object.store.khazana.version>

why do we use a release canditate version of dependency in the released version ?

Reply:
These changes was suggested by you. in this PR.
https://github.com/mosip/registration/pull/1689/files

Reply:
As discussed, did we confirm from @sowmya695 that all the required changes of 1.1.5.5-rc2 is available in 1.1.5.5 ?

Reply:
changed khazana version to 1.1.5.5